P1684-mdarray/P1684.md

```diff
@@ -884,14 +895,130 @@ constexpr mdarray(container_type&& c, const extents_type& ext);
 constexpr mdarray(container_type&& c, const mapping_type& m);
 ```
 
-[20]{.pnum} *Preconditions:* `c.size() == m.required_span_size()` is `true`.
+[20]{.pnum} *Preconditions:* `c.size() >= m.required_span_size()` is `true`.
 
 [21]{.pnum} *Effects:*
 
    * [21.1]{.pnum} Direct-non-list-initializes _`map_`_ with `m`, and
 
    * [22.2]{.pnum} Direct-non-list-initializes `ctr_` with `std::move(c)`.
 
+```c++
+template<class... SizeTypes>
+  explicit constexpr mdarray(std::initializer_list<element_type> init, SizeTypes... exts);
+```
+
+[7]{.pnum} *Constraints:*
+
+   * [7.1]{.pnum} `(is_convertible_v<SizeTypes, size_type> && ...)` is `true`,
+
+   * [7.2]{.pnum} `is_constructible_v<extents_type, static_cast<size_type>(SizeTypes)...>` is `true`,
+
+   * [7.3]{.pnum} `is_constructible_v<mapping_type, extents_type>` is `true`, and
+
+   * [7.4]{.pnum} `is_constructible_v<container_type, std::initializer_list<element_type>>` is `true`.
+
+[8]{.pnum} *Preconditions:* `init.size() >= mapping_type(extents_type(static_cast<size_type>(std::move(exts))...)).required_span_size()` is `true`.
```

Comment:
What would it mean for `init.size()` to be greater than the required span size from `exts...`?  Would that just cut off the "extra" elements of the `initializer_list`?